package.json

```diff
@@ -2,14 +2,22 @@
   "name": "emailcorrect",
   "version": "0.0.1",
   "description": "Test and auto correct e-mail addresses",
-  "main": "index.js",
+  "main": "./src/emailcorrect.js",
   "scripts": {
-    "test": "script/test"
+    "test": "npm run test:lint && npm run test:unit",
+    "test:lint": "eslint src/**/*.js",
+    "test:unit": "node --harmony --harmony_proxies ./node_modules/.bin/istanbul --colors --report=text --report=lcov cover ./node_modules/.bin/_mocha -- \"test/**/*.test.js\" --recursive --harmony_proxies --require ./test/setup.js",
```

Comment:
Why are all these flags needed? Isn't the code compiled to something which doesn't need these flags?